.travis.yml

```diff
@@ -19,7 +20,7 @@ install:
   - bundle install
 script:
   - bundle exec rails webpacker:compile
-  - bundle exec rails spec
+  - RUBYOPT='-W:no-deprecated -W:no-experimental' bundle exec rails spec
```

Comment:
Why this?

Reply:
I added this to hide all of the ruby 2.7 errors that fill the page when rspec gets run. There were other deprecation messages that were not readable because of how many ruby 2.7 deprecations were showing up, so I thought it'd be a good idea to hide the deprecations we can't change for now.

Let me know if you'd prefer that I revert it!

Reply:
This is great thanks. We should remember to upgrade and take it out someday but this is definitely an improvement :)